e2e/test/scenarios/dashboard/reproductions/13736-dashboard-should-work-if-some-cards-are-broken.cy.spec.js

```diff
@@ -0,0 +1,85 @@
+import {
+  getDashboardCards,
+  restore,
+  updateDashboardCards,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+const { ORDERS_ID } = SAMPLE_DATABASE;
+
+const questionDetails1 = {
+  name: "Orders question",
+  query: { "source-table": ORDERS_ID },
+};
+
+const questionDetails2 = {
+  name: "Orders count",
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"]],
+  },
+};
+
+describe("issue 13736", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should work even if some cards are broken (metabase#13736)", () => {
+    cy.createQuestion(questionDetails1, {
+      wrapId: true,
+      idAlias: "questionId1",
+    });
+    cy.createQuestion(questionDetails2, {
+      wrapId: true,
+      idAlias: "questionId2",
+    });
```

Comment:
1. We don't need the questions to be different.
2. I suggest to use more descriptive names:

```suggestion
    cy.createQuestion(questionDetails, {
      wrapId: true,
      idAlias: "failingQuestionId",
    });
    cy.createQuestion(questionDetails, {
      wrapId: true,
      idAlias: "successfulQuestionId",
    });
```

Reply:
Agree. This definitely is a lot better.

Reply:
Fixed in 5a7cb4528c60ae624f7a062b123b230e8e250a7f